cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1531,64 +1531,77 @@ pub(super) mod test {
         };
 
         let expected_string = indoc! {r#"
-            #
-            # This is a Flox environment manifest.
-            # Visit flox.dev/docs/concepts/manifest/
-            # or see flox-edit(1), manifest.toml(5) for more information.
-            #
-            # Flox manifest version managed by Flox CLI
-            version = 1
-
-            # List packages you wish to install in your environment inside
-            # the `[install]` section.
+            ## Flox Environment Manifest ------------------------------------------------
+            ##
+            ##   EVERYTHING you need to know about the MANIFEST is explained here:
+            ##
```

Comment:
the all-caps reads like somebody is yelling at me

I think lowercase, or just completely removing this line would work just fine